File: packages/gunreip/laravel-translation-workbench/resources/views/pages/tw-graph/samples/documentation/idea-to-paper/flow/switch-case/code-examples/switch-action-sequence.php
<?php
$status = 'draft';
$format = 'text';

switch ($status) {
    case 'draft':
    case 'review':
    case 'revision':
        prepareEditing();
        switch ($format) {
            case 'text':
                openTextEditor();
                break;
            case 'image':
                openImageEditor();
                break;
            default:
                openPlainEditor();
                break;
        }
        saveEditingResult();
        break; // Exit the outer switch after the complete action sequence.
    case 'published':
        displayArticle();
        break;
    default:
        showStatusHint();
        break;
}

// Continue after the selected action sequence.
continueProcess();

File: packages/gunreip/laravel-translation-workbench/resources/views/pages/tw-graph/samples/documentation/idea-to-paper/flow/switch-case/code-examples/switch-action-sequence.cs
string status = "draft";
string format = "text";

switch (status) {
    case "draft":
    case "review":
    case "revision":
        PrepareEditing();
        switch (format) {
            case "text":
                OpenTextEditor();
                break;
            case "image":
                OpenImageEditor();
                break;
            default:
                OpenPlainEditor();
                break;
        }
        SaveEditingResult();
        break; // Exit the outer switch after the complete action sequence.
    case "published":
        DisplayArticle();
        break;
    default:
        ShowStatusHint();
        break;
}

// Continue after the selected action sequence.
ContinueProcess();
